spec/expression.dd

```diff
@@ -110,13 +110,18 @@ $(GRAMMAR
 $(GNAME ConditionalExpression):
     $(GLINK OrOrExpression)
     $(GLINK OrOrExpression) $(D ?) $(GLINK Expression) $(D :) $(I ConditionalExpression)
+    $(GLINK OrOrExpression) $(D ?) $(I IfTrue)$(OPT) $(D :) $(I ConditionalExpression)
+    $(GNAME IfTrue):
+       $(EXPRESSION)
 )
 
     $(P The first expression is converted to $(D bool), and is evaluated.
     )
 
-    $(P If it is $(D true), then the second expression is evaluated, and
-        its result is the result of the conditional expression.
+    $(P If it is $(D true) and  the second expression is provided, then
```

Comment:
There are two spaces between `and` and `the`.

Reply:
@jacob-carlborg don't forget you can edit PRs directly for such things (I'll do it just now)

Reply:
I haven't tested this but the documentation says:

"You can only make commits on pull request branches that: are opened in a repository that you have push access to and that were created from a fork of that repository" [1]

[1] https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/